.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - "node"
+  - "stable"
```

Comment:
Used to put `"stable"` but changed to `"node"` as it's what they use in https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions (not a blocker though :))